deploy/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  deploy:
+    image: allofustest/workbench:buildimage-0.0.13
+    entrypoint: /bootstrap-docker.sh
+    user: circleci
+    environment:
+      # Required: a GitHub release git tag, e.g. v1-0-rc1
+      - DATA_BROWSER_VERSION
+    volumes:
+      # Cache the codebase and gradle for deployment only.
+      - gradle-cache:/.gradle
+      - data-browser:/home/circleci/workbench
```

Comment:
Not sure if /home/circle/data-browser is present. 
Do i need to change this ?

Reply:
Yes - 'data-browser' will be there, this is the name of the cloned github directory.